CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.3] - 2019-07-26
```

Comment:
I'd like to leave it in unreleased. Let's change that in the release PR.

Reply:
done